diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3aa5ef8..f3b3d53 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -397,7 +397,7 @@ set( _EXEDIR "${_DEST}" )
 # Setup RPATH handling
 set( CMAKE_BUILD_RPATH "${_DEST}/${_PKGLIB}" )
 set( CMAKE_BUILD_WITH_INSTALL_RPATH FALSE )
-set( CMAKE_INSTALL_RPATH "$ORIGIN/../${_PKGLIB}" )
+set( CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_FULL_LIBDIR}/audacity" )
 set( CMAKE_INSTALL_RPATH_USE_LINK_PATH FALSE )
 
 # Adjust them for the Mac
diff --git a/scripts/build/linux/fix_rpath.cmake b/scripts/build/linux/fix_rpath.cmake
index ed23f2b..6036f6b 100644
--- a/scripts/build/linux/fix_rpath.cmake
+++ b/scripts/build/linux/fix_rpath.cmake
@@ -6,7 +6,7 @@ set(private_lib_dir "${install_dir}/${_LIBDIR}/audacity")
 
 message(STATUS "Running fix_rpath in ${private_lib_dir}")
 
-file(GLOB libs LIST_DIRECTORIES Off "${private_lib_dir}/*.so*")
+file(GLOB_RECURSE libs LIST_DIRECTORIES Off "${private_lib_dir}/*.so*")
 
 function(get_rpath outvar lib)
    execute_process(
@@ -47,11 +47,11 @@ foreach(lib ${libs})
    get_rpath(current_rpath ${lib})
 
    if( current_rpath )
-      if(NOT current_rpath STREQUAL "$ORIGIN")
+      if(NOT current_rpath STREQUAL "${_PKGLIB}")
          file(RPATH_CHANGE
                FILE "${lib}"
                OLD_RPATH "${current_rpath}"
-               NEW_RPATH "$ORIGIN")
+               NEW_RPATH "${_PKGLIB}")
       endif()
    else()
       message(STATUS "\tNo rpath was set")
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 7b90c1b..942dffb 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1630,6 +1630,7 @@ else()
                FILES_MATCHING PATTERN "*.so*" )
 
       install( CODE "set( _LIBDIR \"${_LIBDIR}\")" )
+      install( CODE "set( _PKGLIB \"${CMAKE_INSTALL_FULL_LIBDIR}/audacity\")" )
       install( SCRIPT "${CMAKE_SOURCE_DIR}/scripts/build/linux/fix_rpath.cmake" )
 
       install( FILES "${_INTDIR}/audacity.desktop"
diff --git a/modules/CMakeLists.txt b/modules/CMakeLists.txt
index d464383..0da2a9c 100644
--- a/modules/CMakeLists.txt
+++ b/modules/CMakeLists.txt
@@ -20,6 +20,11 @@ if( NOT CMAKE_SYSTEM_NAME MATCHES "Darwin" )
    if( NOT "${CMAKE_GENERATOR}" MATCHES "Visual Studio*")
       install( DIRECTORY "${_DEST}/modules"
                DESTINATION "${_PKGLIB}" )
+
+      install( CODE "set( _LIBDIR \"${_LIBDIR}\")" )
+      install( CODE "set( _PKGLIB \"${CMAKE_INSTALL_FULL_LIBDIR}/audacity\")" )
+      install( SCRIPT "${CMAKE_SOURCE_DIR}/scripts/build/linux/fix_rpath.cmake" )
+
    endif()
 endif()
 
