--- ./src/AudioIO.cpp.orig	2017-12-03 17:58:41.137697095 +0000
+++ ./src/AudioIO.cpp	2017-12-03 18:02:25.447946534 +0000
@@ -451,8 +451,8 @@ writing audio.
    #endif
    #define ROUND(x) (int) ((x)+0.5)
    //#include <string.h>
-   #include "../lib-src/portmidi/pm_common/portmidi.h"
-   #include "../lib-src/portaudio-v19/src/common/pa_util.h"
+   #include "portmidi.h"
+   #include "../lib-src/portaudio-v19/src/common/pa_util.h"
    #include "NoteTrack.h"
 #endif
 
