diff -Nur audacity-src-1.3.9-orig/src/AudioIO.cpp audacity-src-1.3.9/src/AudioIO.cpp
--- audacity-src-1.3.9-orig/src/AudioIO.cpp	2009-08-31 12:45:38.000000000 +0200
+++ audacity-src-1.3.9/src/AudioIO.cpp	2009-12-04 14:40:41.559205870 +0100
@@ -2750,7 +2750,7 @@
 
    // Stop recording if 'silence' is detected
    if(gAudioIO->mPauseRec && inputBuffer) {
-      if(gAudioIO->mInputMeter->GetMaxPeak() < gAudioIO->mSilenceLevel ) {
+      if(gAudioIO->mInputMeter && (gAudioIO->mInputMeter->GetMaxPeak() < gAudioIO->mSilenceLevel) ) {
          if(!gAudioIO->IsPaused()) {
             AudacityProject *p = GetActiveProject();
             wxCommandEvent dummyEvt;
