diff -ur audacity-src-1.3.12-beta.orig/src/widgets/ASlider.cpp audacity-src-1.3.12-beta/src/widgets/ASlider.cpp
--- audacity-src-1.3.12-beta.orig/src/widgets/ASlider.cpp	2010-03-30 22:28:31.000000000 +1100
+++ audacity-src-1.3.12-beta/src/widgets/ASlider.cpp	2010-10-04 19:22:25.000000000 +1100
@@ -397,7 +397,7 @@
       stepValue = STEP_CONTINUOUS;
       break;
    case SPEED_SLIDER:
-      minValue = 0.01f;
+      minValue = 0.09f; // libsamplerate lower limit is 1/12
       maxValue = 3.0f;
       stepValue = STEP_CONTINUOUS;
       break;
