diff -Nur audacity-src-1.3.6-orig/src/import/ImportFLAC.cpp audacity-src-1.3.6/src/import/ImportFLAC.cpp
--- audacity-src-1.3.6-orig/src/import/ImportFLAC.cpp	2008-10-25 14:14:27.000000000 +0200
+++ audacity-src-1.3.6/src/import/ImportFLAC.cpp	2008-11-05 00:15:33.000000000 +0100
@@ -361,6 +361,8 @@
               Tags *tags)
 {
    wxASSERT(mStreamInfoDone);
+   if (!mStreamInfoDone)
+      return eImportFailed;
 
    CreateProgress();
 
