--- ./src/effects/ladspa/LadspaEffect.cpp.orig	2019-10-22 07:29:41.000000000 +1100
+++ ./src/effects/ladspa/LadspaEffect.cpp	2019-11-10 09:44:41.865684664 +1100
@@ -378,11 +378,10 @@
 #else
 
    pathList.push_back(wxGetHomeDir() + wxFILE_SEP_PATH + wxT(".ladspa"));
+   pathList.push_back(wxT("/usr/%{_lib}/ladspa"));
 #if defined(__LP64__)
-   pathList.push_back(wxT("/usr/local/lib64/ladspa"));
    pathList.push_back(wxT("/usr/lib64/ladspa"));
 #endif
-   pathList.push_back(wxT("/usr/local/lib/ladspa"));
    pathList.push_back(wxT("/usr/lib/ladspa"));
    pathList.push_back(wxT(LIBDIR) wxT("/ladspa"));
 
