diff -Nur audacity-src-1.3.7-orig/configure audacity-src-1.3.7/configure
--- audacity-src-1.3.7-orig/configure	2009-01-27 21:50:59.000000000 +0100
+++ audacity-src-1.3.7/configure	2009-02-28 16:06:02.000000000 +0100
@@ -7774,12 +7774,12 @@
     pkg_cv_VAMP_CFLAGS="$VAMP_CFLAGS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { ($as_echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"vamp-hostsdk >= 2.0\"") >&5
-  ($PKG_CONFIG --exists --print-errors "vamp-hostsdk >= 2.0") 2>&5
+    { ($as_echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"vamp-hostsdk >= 1.1.0\"") >&5
+  ($PKG_CONFIG --exists --print-errors "vamp-hostsdk >= 1.1.0") 2>&5
   ac_status=$?
   $as_echo "$as_me:$LINENO: \$? = $ac_status" >&5
   (exit $ac_status); }; then
-  pkg_cv_VAMP_CFLAGS=`$PKG_CONFIG --cflags "vamp-hostsdk >= 2.0" 2>/dev/null`
+  pkg_cv_VAMP_CFLAGS=`$PKG_CONFIG --cflags "vamp-hostsdk >= 1.1.0" 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -7790,12 +7790,12 @@
     pkg_cv_VAMP_LIBS="$VAMP_LIBS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { ($as_echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"vamp-hostsdk >= 2.0\"") >&5
-  ($PKG_CONFIG --exists --print-errors "vamp-hostsdk >= 2.0") 2>&5
+    { ($as_echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"vamp-hostsdk >= 1.1.0\"") >&5
+  ($PKG_CONFIG --exists --print-errors "vamp-hostsdk >= 1.1.0") 2>&5
   ac_status=$?
   $as_echo "$as_me:$LINENO: \$? = $ac_status" >&5
   (exit $ac_status); }; then
-  pkg_cv_VAMP_LIBS=`$PKG_CONFIG --libs "vamp-hostsdk >= 2.0" 2>/dev/null`
+  pkg_cv_VAMP_LIBS=`$PKG_CONFIG --libs "vamp-hostsdk >= 1.1.0" 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -7813,9 +7813,9 @@
         _pkg_short_errors_supported=no
 fi
         if test $_pkg_short_errors_supported = yes; then
-	        VAMP_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors "vamp-hostsdk >= 2.0" 2>&1`
+	        VAMP_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors "vamp-hostsdk >= 1.1.0" 2>&1`
         else
-	        VAMP_PKG_ERRORS=`$PKG_CONFIG --print-errors "vamp-hostsdk >= 2.0" 2>&1`
+	        VAMP_PKG_ERRORS=`$PKG_CONFIG --print-errors "vamp-hostsdk >= 1.1.0" 2>&1`
         fi
 	# Put the nasty error message in config.log where it belongs
 	echo "$VAMP_PKG_ERRORS" >&5
diff -Nur audacity-src-1.3.7-orig/src/effects/vamp/VampEffect.cpp audacity-src-1.3.7/src/effects/vamp/VampEffect.cpp
--- audacity-src-1.3.7-orig/src/effects/vamp/VampEffect.cpp	2009-01-27 21:50:58.000000000 +0100
+++ audacity-src-1.3.7/src/effects/vamp/VampEffect.cpp	2009-02-28 15:45:52.000000000 +0100
@@ -13,9 +13,9 @@
 
 #include "VampEffect.h"
 
-#include <vamp-hostsdk/Plugin.h>
-#include <vamp-hostsdk/PluginChannelAdapter.h>
-#include <vamp-hostsdk/PluginInputDomainAdapter.h>
+#include <vamp-sdk/Plugin.h>
+#include <vamp-sdk/hostext/PluginChannelAdapter.h>
+#include <vamp-sdk/hostext/PluginInputDomainAdapter.h>
 
 #include <wx/wxprec.h>
 #include <wx/button.h>
@@ -311,20 +311,12 @@
       Vamp::RealTime ftime0 = fli->timestamp;
       double ltime0 = ftime0.sec + (double(ftime0.nsec) / 1000000000.0);
 
-      Vamp::RealTime ftime1 = ftime0;
-      if (fli->hasDuration) ftime1 = ftime0 + fli->duration;
-      double ltime1 = ftime1.sec + (double(ftime1.nsec) / 1000000000.0);
-
       wxString label = LAT1CTOWX(fli->label.c_str());
       if (label == wxString()) {
-         if (fli->values.empty()) {
             label = wxString::Format(LAT1CTOWX("%.3f"), ltime0);
-         } else {
-            label = wxString::Format(LAT1CTOWX("%.3f"), *fli->values.begin());
-         }
       }
       
-      ltrack->AddLabel(ltime0, ltime1, label);
+      ltrack->AddLabel(ltime0, ltime0, label);
    }
 }
 
diff -Nur audacity-src-1.3.7-orig/src/effects/vamp/VampEffect.h audacity-src-1.3.7/src/effects/vamp/VampEffect.h
--- audacity-src-1.3.7-orig/src/effects/vamp/VampEffect.h	2009-01-27 21:50:58.000000000 +0100
+++ audacity-src-1.3.7/src/effects/vamp/VampEffect.h	2009-02-28 15:45:52.000000000 +0100
@@ -22,7 +22,7 @@
 
 #include <wx/dialog.h>
 
-#include <vamp-hostsdk/PluginLoader.h>
+#include <vamp-sdk/hostext/PluginLoader.h>
 
 void LoadVampPlugins();
 
