--- audacity-src-1.3.12-beta.orig/src/FFmpeg.cpp	2010-03-30 22:28:54.000000000 +1100
+++ audacity-src-1.3.12-beta/src/FFmpeg.cpp	2010-11-10 22:06:29.000000000 +1100
@@ -204,7 +204,7 @@
    return ret;
 }
 
-static int ufile_write(URLContext *h, unsigned char *buf, int size)
+static int ufile_write(URLContext *h, const unsigned char *buf, int size)
 {
    return (int) ((wxFile *) h->priv_data)->Write(buf, size);
 }
