diff -Nur audacity-src-1.3.4-beta-orig/src/effects/ladspa/LoadLadspa.cpp audacity-src-1.3.4-beta/src/effects/ladspa/LoadLadspa.cpp
--- audacity-src-1.3.4-beta-orig/src/effects/ladspa/LoadLadspa.cpp	2008-01-23 18:32:52.000000000 +0100
+++ audacity-src-1.3.4-beta/src/effects/ladspa/LoadLadspa.cpp	2008-03-21 23:21:45.000000000 +0100
@@ -92,8 +92,8 @@
 
    #ifdef __WXGTK__
    wxGetApp().AddUniquePathToPathList(wxT(INSTALL_PREFIX) wxT("/ladspa"), pathList);
-   wxGetApp().AddUniquePathToPathList(wxT("/usr/local/lib/ladspa"), pathList);
-   wxGetApp().AddUniquePathToPathList(wxT("/usr/lib/ladspa"), pathList);
+   wxGetApp().AddUniquePathToPathList(wxT("/usr/local/__RPM_LIB__/ladspa"), pathList);
+   wxGetApp().AddUniquePathToPathList(wxT("__RPM_LIBDIR__/ladspa"), pathList);
    #endif
 
    #ifdef __WXMAC__
diff -Nur audacity-src-1.3.5-beta-orig/lib-src/libvamp/vamp-sdk/PluginHostAdapter.cpp audacity-src-1.3.5-beta/lib-src/libvamp/vamp-sdk/PluginHostAdapter.cpp
--- audacity-src-1.3.5-beta-orig/lib-src/libvamp/vamp-sdk/PluginHostAdapter.cpp	2008-05-07 23:19:58.000000000 +0200
+++ audacity-src-1.3.5-beta/lib-src/libvamp/vamp-sdk/PluginHostAdapter.cpp	2008-05-11 20:56:59.000000000 +0200
@@ -75,7 +75,7 @@
 #ifdef __APPLE__
 #define DEFAULT_VAMP_PATH "$HOME/Library/Audio/Plug-Ins/Vamp:/Library/Audio/Plug-Ins/Vamp"
 #else
-#define DEFAULT_VAMP_PATH "$HOME/vamp:$HOME/.vamp:/usr/local/lib/vamp:/usr/lib/vamp"
+#define DEFAULT_VAMP_PATH "$HOME/vamp:$HOME/.vamp:/usr/local/__RPM_LIB__/vamp:__RPM_LIBDIR__/vamp"
 #endif
 #endif
 
