--- audacity-src-2.0.1/src/audacity.desktop.in	2012-06-26 07:27:46.000000000 +1000
+++ audacity-src-2.0.0/src/audacity.desktop.in	2012-07-04 22:11:31.000000000 +1000
@@ -5,16 +5,17 @@
 GenericName=Sound Editor
 GenericName[de]=Audio-Editor
 GenericName[ru]=Редактор звуковых файлов
-Comment=Record and edit audio files
+Comment=Record and edit audio files with multitrack capability
 Comment[de]=Audio-Dateien aufnehmen und bearbeiten
 Comment[ru]=Запись и редактирование звуковых файлов
 
 Icon=@AUDACITY_NAME@
 
 Type=Application
-Categories=AudioVideo;Audio;AudioVideoEditing;
+Categories=AudioVideo;Audio;GTK;GNOME;Sequencer;X-Jack;AudioVideoEditing;
 
 Exec=@AUDACITY_NAME@ %F
-StartupNotify=false
+StartupNotify=true
+StartupWMClass=Audacity
 Terminal=false
-MimeType=application/ogg;audio/basic;audio/mpeg;audio/x-aiff;audio/x-mp3;audio/x-wav;application/x-audacity-project;
+MimeType=application/ogg;audio/basic;audio/mpeg;audio/x-aiff;audio/x-aiffc;audio/x-aifc;audio/x-mp3;audio/x-wav;application/x-audacity-project;audio/midi;audio/x-flac;
