--- audacity-src-2.0.3/src/FFmpeg.h	2013-01-18 16:50:58.000000000 +0100
+++ audacity-src-2.0.3.new/src/FFmpeg.h	2013-05-04 17:39:13.754469535 +0200
@@ -38,11 +38,17 @@
 
    #include <libavcodec/avcodec.h>
    #include <libavformat/avformat.h>
+   #include <libavformat/avio.h>
    #include <libavutil/fifo.h>
    #include <libavutil/mathematics.h>
 
    #if LIBAVFORMAT_VERSION_INT < AV_VERSION_INT(52, 102, 0)
    #define AVIOContext ByteIOContext
+   #if defined(DISABLE_DYNAMIC_LOADING_FFMPEG)
+   #define avio_read get_buffer
+   #define avio_seek url_fseek
+   #define avio_close url_fclose
+   #endif
    #endif
 
    #if LIBAVCODEC_VERSION_INT < AV_VERSION_INT(52, 94, 1)
