diff -Nur audacity-src-1.3.2-beta-orig/src/xml/XMLFileReader.cpp audacity-src-1.3.2-beta/src/xml/XMLFileReader.cpp
--- audacity-src-1.3.2-beta-orig/src/xml/XMLFileReader.cpp	2006-10-29 01:05:26.000000000 +0200
+++ audacity-src-1.3.2-beta/src/xml/XMLFileReader.cpp	2007-08-28 12:11:08.000000000 +0200
@@ -61,9 +61,9 @@
       size_t len = fread(buffer, 1, bufferSize, theXMLFile.fp());
       done = (len < bufferSize);
       if (!XML_Parse(mParser, buffer, len, done)) {
-         mErrorStr.Printf(_("Error: %hs at line %d"),
+         mErrorStr.Printf(_("Error: %hs at line %lu"),
 			  XML_ErrorString(XML_GetErrorCode(mParser)),
-			  XML_GetCurrentLineNumber(mParser));
+                          (long unsigned int)XML_GetCurrentLineNumber(mParser));
          theXMLFile.Close();
          return false;
       }
